src/coreclr/src/System.Private.CoreLib/src/System/Reflection/RuntimeExceptionHandlingClause.cs

```diff
@@ -38,6 +39,11 @@ public override int FilterOffset
 
         public override Type? CatchType
         {
+            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
```

Comment:
Should we mark `MethodBody` -> `ExceptionHandlingCaluse` -> ... and friends as `RequiresUnreferencedCode` instead?

My line of thinking is that we want to warn whenever trimming can produce different results than pre-trimming. Since linker is going to potentially stub out method bodies, replace basic blocks with nops, etc. maybe these should all warn?

Reply:
It seem marking `MethodBase.GetMethodBody()` with `RequiresUnreferencedCode` would be enough - it should cover changes to IL, local variables, exception clauses, ... As far as I can tell all of those have to start with somehow getting the `MethodBody` object, which seems to be only possible via `MethodBase.GetMethodBody`.